.github/workflows/release.yml

```
@@ -0,0 +1,77 @@
+name: release
+
+on:
+    workflow_dispatch:
+        inputs:
+            RELEASE_VERSION:
+                description: 'Version to release'
+                required: true
+
+jobs:
+    release:
+        name: Release ${{ github.event.inputs.RELEASE_VERSION }}
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v4
+            
+            - name: Install uv
+              uses: astral-sh/setup-uv@v5
+
+            - name: Install dependencies
+              run: |
+                uv venv --python=3.11
+                uv pip install -r requirements/ci.txt -r pipeline/requirements.txt
+
+            - name: Download raw data
+              run: |
+                uv run make download-raw
+
+            - name: Process raw data into duckdb file
+              run: |
+                uv run make process
+
+            - name: Set environment variable for version
+              run: |
+                echo "VERSION_UNDERSCORE=${{ github.event.inputs.RELEASE_VERSION }}" | sed 's/\./_/g' >> $GITHUB_ENV
+            
+            - name: Make sure file exists
+              run: |
+                if [ ! -f data/processed/scipeds_${{ env.VERSION_UNDERSCORE }}.duckdb ]; then
```

```suggestion
                if [ ! -f data/processed/scipeds_${VERSION_UNDERSCORE}.duckdb ]; then
```
I've seen tips that it's a little safer/more reliable to use normal bash environment variable expansion for environment variables rather than the GitHub Actions templating injection, when trying to reference in a script. This is minor and probably doesn't make much of a difference.